src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownEventListener.java

```diff
@@ -358,9 +359,15 @@ public void onTownStatusScreen(TownStatusScreenEvent event) {
 				out.add(Translation.of("status_town_siege_defender", siege.getDefenderNameForDisplay()));
 
 				switch (siegeStatus) {
-	                case IN_PROGRESS:
-						// > Balance: 530
-						out.add(Translation.of("status_town_siege_status_siege_balance", siege.getSiegeBalance()));
+					case IN_PROGRESS:
+						// > Balance: 530 | Pending: +130
+						String balanceLine = Translation.of("status_town_siege_status_siege_balance", siege.getSiegeBalance());
+						// If the session is active with points add the " | Pending: +130"

```

Comment:
:construction: 
As per previous comment this comment should read something like:
```
// If the battle is active add the " | Pending: +130"
```